R/get_bad_bambooIDs.R

```diff
@@ -0,0 +1,17 @@
+#' Bamboo API get request wrapper
```

Comment:
This function makes a ton of sense and I'd like to see it used in dataPipeline, but it should be located in dataPipeline instead of bambooR. bambooR is intended to be a generic Bamboo wrapper whereas this function is specific to Propeller.

Reply:
Or rather, perhaps this belongs in propUtils, now that I think about it...